lib/ansible/modules/files/lineinfile.py

```diff
@@ -248,15 +252,37 @@ def present(module, dest, regexp, line, insertafter, insertbefore, create,
             try:
                 os.makedirs(b_destpath)
```

Comment:
Once the locking code is updated to raise IOError if the file does not exist, we'll have to create the file here.  I believe we can do that like:
``` python
open(b_dest, 'ab').close()
```

I believe using append mode, if the file doesn't exist, it will be created.  If the file does exist before the filehandle is closed, it won't be modified.

Reply:
Verified this is the case: an existing file is not modified but an empty file is created if it does not exist using the above code.